prime-router/docs/getting-started/standard-operating-procedures/observation-mapping-table.md

```diff
@@ -0,0 +1,77 @@
+# How to Update the Observation Mapping Table
+
+## Introduction
+
+The observation mapping table maps a plethora of conditions, observations, and aoe question to codes in various code
+systems. The data for this table comes from multiple public sources. Once loaded in the database, the prime CLI can be
+used to poll public RESTful sources for updated data.
+
+A local csv file is kept both as a reference and to seed the local development environment. It should be updated
+periodically by dumping the table from staging/prod or using the CLI to update the local table.
+
+## Update prod/staging tables using RESTful sources
+TBD
+
+## Update the local CSV by getting tables from prod/staging
+Use the prime CLI `lookuptables get` command:  
+Change the `--env` value as needed
+```zsh
+prime lookuptables get -n observation-mapping -v 1 -o observation-mappings.csv --env prod
+```
+
+## Create or update prod/staging tables using a local CSV
+Use the prime CLI `lookuptables create` command:
+```zsh
+./prime lookuptables create prod -i ./observation-mapping.csv -a -n observation-mapping
+```
+
+## Reload table locally
+Run the gradle task `reloadTables`:
+```zsh
+./gradlew reloadTables
+```
+
+## Creating a CSV from data sources
+
+### Columns
+Refer to the latest [observation-mapping.csv](/prime-router/metadata/tables/local/observation-mappings.csv)
+use the following columns at your peril:
```

Comment:
😄 Can we elaborate on what peril means here?  Is this just saying be careful creating a CSV be hand

Reply:
Main peril I was consider was using the columns listed in this markdown doc and failing to check the CSV.
(i.e. CSV gets updated, has new columns, markdown update is forgotten).